tracer/src/Datadog.Trace.Coverage.collector/CoverageCollector.cs

```diff
@@ -0,0 +1,253 @@
+// <copyright file="CoverageCollector.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using System.Xml;
+using Datadog.Trace.Ci.Configuration;
+using Datadog.Trace.ClrProfiler;
+using Datadog.Trace.Vendors.Newtonsoft.Json;
+using Datadog.Trace.Vendors.Newtonsoft.Json.Linq;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection;
+using Formatting = Datadog.Trace.Vendors.Newtonsoft.Json.Formatting;
+
+#pragma warning disable SA1300
```

Comment:
```suggestion
// Disable SA1300 Warning due the `collector` lowercase in the namespace
#pragma warning disable SA1300
```

Reply:
done in [8544fbd](https://github.com/DataDog/dd-trace-dotnet/pull/2739/commits/8544fbd8c6c5425101db2dbf9ae5de5fc4f8aad0)